guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -58,6 +58,15 @@ ifdef::katello[]
 # {package-manager} config-manager --set-enabled powertools
 ----
 endif::[]
+ifdef::foreman-el,katello[]
++
+. Enable the PostgreSQL 12 module:
```

Comment:
I wonder if we need some reference documentation on what to do if a different module is already enabled etc.

Reply:
Added a note